src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,178 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    static class Node<T> {
+        private T data;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T data, Node<T> next) {
```

Comment:
```suggestion
    private static class Node<T> {
        private T data;
        private Node<T> next;
        private Node<T> prev;

        Node(Node<T> prev, T data, Node<T> next) {
```
place it at the end of the file